src/nat/cli/commands/workflow/workflow_commands.py

```
@@ -172,6 +171,9 @@ def create_command(workflow_name: str, install: bool, workflow_dir: str, descrip
         description (str): Description to pre-popluate the workflow docstring.
     """
     try:
+        if not workflow_name.strip():
+            raise click.ClickException("Workflow name cannot be empty.")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)